es-app/src/SystemData.cpp

```diff
@@ -60,7 +60,7 @@ void SystemData::setIsGameSystemStatus()
 	// we exclude non-game systems from specific operations (i.e. the "RetroPie" system, at least)
 	// if/when there are more in the future, maybe this can be a more complex method, with a proper list
 	// but for now a simple string comparison is more performant
-	mIsGameSystem = (mName != "retropie");
+	mIsGameSystem = (mName != "retropie") && (mName != "kodi");
```

Comment:
Aha I see. Well it's less of an issue then. We can always put it back if users have an issue with it, but it's not a big problem. Will merge this as is for now. thanks.